src/components/Graph.react.js

```diff
@@ -21,20 +21,44 @@ const filterEventData = (gd, eventData, event) => {
             const pointData = filter(function(o) {
                 return !contains(type(o), ['Object', 'Array'])
             }, fullPoint);
-
-            if (has('customdata', data[pointData.curveNumber]) &&
+            if (has('curveNumber', fullPoint) &&
                 has('pointNumber', fullPoint) &&
-                has('curveNumber', fullPoint)
-            ) {
-                pointData['customdata'] = data[
-                    pointData.curveNumber
-                ].customdata[fullPoint.pointNumber];
+                has('customdata', data[pointData.curveNumber])) {
+               pointData['customdata'] = data[
+                                            pointData.curveNumber
+                                        ].customdata[fullPoint.pointNumber];
```

Comment:
this makes sense to me. can you format this to match the style of the rest of the code? In particular, the indenting needs to be updated